scripts/dashboard_dev_docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# Builds both the node's and the analysis dashboard and runs pkger afterwards.
+
+echo "::: Building /plugins/dashboard/frontend :::"
+docker run -it --rm \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --volume="/etc/group:/etc/group:ro" \
+    --volume="/etc/passwd:/etc/passwd:ro" \
+    --volume="/etc/shadow:/etc/shadow:ro" \
+    -v $(pwd):/tmp/mnt \
+    -e YARN_CACHE_FOLDER=/tmp/ \
+    -e HOME=/tmp/ \
+    -w /tmp/mnt/plugins/dashboard/frontend node:12.16 bash -c "yarn install"
```

Comment:
This should work too no?
```suggestion
    -w /tmp/mnt/plugins/dashboard/frontend node:12.16 bash -c "yarn install && yarn start"
```